src/main/java/com/spotinst/sdkjava/model/bl/ocean/ecs/ClusterOptimizeImageConfiguration.java

```diff
@@ -7,13 +7,12 @@
 import java.util.Set;
 
 public class ClusterOptimizeImageConfiguration {
-
+    @JsonIgnore
     private Set<String>  isSet;
     private Boolean      shouldOptimizeEcsAmi;
     private String       performAt;
     private List<String> timeWindows;
```

Comment:
Please reconfirm if it should be list of strings.

Reply:
Checked for Elastigroup and Ocean Kubernetes Cluster - timewindows parameter is list of String.